data/human/hails.txt

```diff
@@ -11,6 +11,57 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+
+phrase "bribe rejection hail"
+	word
+		"It is too late: "
+		"Your attempt is futile; "
+	word
+		"we are too honorable to accept this from an enemy."
+		"we value honor over greed."
+		"we will not be swayed so easily."
+
+phrase "bribe rejection hail"
+	word
+		"We're "
+	word
+		"fighting a "
+		"in a "
+		"in the middle of a "
+		"trying to win a "
+		"winning a "
+	word
+		"battle"
+		"combat"
+		"conflict"
+		"engagement"
```

Comment:
It should be "an engagement", not "a engagement". A replacement should be able to do this, too.